mne_bids/path.py

```diff
@@ -345,6 +362,17 @@ def copy(self):
         """
         return deepcopy(self)
 
+    def mkdir(self, exist_ok=True, parents=True):
```

Comment:
```suggestion
    def mkdir(self, exist_ok=True, parents=True):
```
Do we even need `exist_ok`?

Also I would propose to rename `parents` to `create_parents`, WDYT?

Reply:
> Do we even need `exist_ok`?

I suppose no. Is there any reason we shouldn't always pass exist_ok=True? I'm game to remove it.

> Also I would propose to rename `parents` to `create_parents`, WDYT?

Hmm I was keeping in line with `pathlib.Path`, but think `create_parents` is more explicit. I'm okay with either one, but I would prefer keeping in line w/ `pathlib` semantics to make it easier to remember.

Reply:
Lmk what you think.

Reply:
please stick to pathlib names as is right now

Reply:
I would just do:

mkdir(parents=False, exist_ok=False)

for follow the param order from pathlib.Path

Reply:
@agramfort 
> please stick to pathlib names as is right now
> mkdir(parents=False, exist_ok=False)
> 
> for follow the param order from pathlib.Path

I don't think we'll need `parents`, no?

Reply:
I guess we can always make the parents. I don't see a reason why not(?) I'll adjust it.